docs-ref-conceptual/eclipse/java-sdk-azure-get-started-eclipse.md

```diff
@@ -0,0 +1,415 @@
+            final String sqlServerName = SdkContext.randomResourceName("sql",10);
+            final String sqlDbName = SdkContext.randomResourceName("dbname",8);
+            final String dbPassword = "YOUR_PASSWORD_HERE";
+
+
+            SqlServer sampleSQLServer = azure.sqlServers().define(sqlServerName)
+                            .withRegion(Region.US_EAST)
+                            .withNewResourceGroup("sampleSqlResourceGroup")
+                            .withAdministratorLogin(adminUser)
+                            .withAdministratorPassword(dbPassword)
+                            .withNewFirewallRule("0.0.0.0","255.255.255.255")
+                            .create();
+
+            SqlDatabase sampleSQLDb = sampleSQLServer.databases().define(sqlDbName).create();
+
+            // assemble the connection string to the database
+            final String domain = sampleSQLServer.fullyQualifiedDomainName();
+            String url = "jdbc:sqlserver://"+ domain + ":1433;" +
+                    "database=" + sqlDbName +";" +
+                    "user=" + adminUser+ "@" + sqlServerName + ";" +
+                    "password=" + dbPassword + ";" +
+                    "encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30;";
+
+            // connect to the database, create a table and insert a entry into it
+            Connection conn = DriverManager.getConnection(url);
+
+            String createTable = "CREATE TABLE CLOUD ( name varchar(255), code int);";
+            String insertValues = "INSERT INTO CLOUD (name, code ) VALUES ('Azure', 1);";
+            String selectValues = "SELECT * FROM CLOUD";
+            Statement createStatement = conn.createStatement();
+            createStatement.execute(createTable);
+            Statement insertStatement = conn.createStatement();
+            insertStatement.execute(insertValues);
+            Statement selectStatement = conn.createStatement();
+            ResultSet rst = selectStatement.executeQuery(selectValues);
+
+            while (rst.next()) {
+                System.out.println(rst.getString(1) + " "
+                        + rst.getString(2));
+            }
+
+
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            System.out.println(e.getStackTrace().toString());
+        }
+    }
+```
+Run the sample from the command line:
+
+Then clean up the resources using the CLI:
+
+```azurecli-interactive
+az group delete --name sampleSqlResourceGroup
+```
+
+## Write a blob into a new storage account
+
+Replace the current main method in `AzureApp.java` with the code below. This code creates an [Azure storage account](https://docs.microsoft.com/azure/storage/storage-introduction) and then uses the Azure Storage libraries for Java to create a new text file in the cloud.
+
+```java
+    public static void main(String[] args) {
```

Comment:
Since this is in the code block, remove the single indent, so that the outermost statements are flush with the left.

Reply:
Otherwise it will render as indented - the code block is rendered as-is, like the HTML code block.